examples/gallery/symbols/bars.py

```diff
@@ -0,0 +1,71 @@
+r"""
+Vertical or horizontal bars
+---------------------------
+The :meth:`pygmt.Figure.plot` method can plot vertical (**b**) or
+horizontal (**B**) bars by passing the corresponding shortcut to
+the ``style`` parameter. By default, *base* = 0 meaning that the
+bar is starting from 0. Append **+b**\[*base*] to change this
+value. To plot multi-band bars, please append
+**+v**\|\ **i**\ *ny* (for verticals bars) or **+v**\|\ **i**\ *nx*
+(for horizontal ones), where *ny* or *nx* indicate the total
+number of bands in the bar (and hence the number of values required
+to follow the *x,y* coordinate pair in the input). Here, **+i**
+means we must accumulate the bar values from the increments
+*dy* or *dx*, while **+v** means we get the complete values relative
+to base. Normally, the bands are plotted as sections of a final
+single bar. Use **+s** to instead split the bar into *ny* or *nx*
+side-by-side, individual and thinner bars. Multi-band bars require
+``cmap=True`` with one color per band.
+"""
+
+import pygmt
+
+fig = pygmt.Figure()
+
+pygmt.makecpt(cmap="roma", series=[0, 4, 1])
+
+with fig.subplot(
+    nrows=2,
+    ncols=2,
+    subsize=("8c", "4c"),
+    frame=["ag"],
```

Comment:
These square brackets are not wrong, but strictly speaking not needed.

```suggestion
    frame="ag",
```